src/actions/delete.js

```diff
@@ -1,24 +1,25 @@
 'use strict';
 require('log-prefix')(() => `[${new Date().toISOString()}] Doctor-core: %s`);
-const {type} = require('ramda');
+const { type } = require('ramda');
 const loadAccount = require('../util/loadAccount');
-const {startSpinner, stopSpinner} = require('../util/spinner');
-const {removeCancelledJobId} = require('../events/cancelled-job');
+const { startSpinner, stopSpinner } = require('../util/spinner');
```

Comment:
If possible, move these spinners to the CLI code.

Reply:
Yes, will do this in separate PR when `doctor-cli` will refer to `doctor-core`